pkg/tasks/alertmanager.go

```diff
@@ -178,6 +178,17 @@ func (t *AlertmanagerTask) create(ctx context.Context) error {
 			return errors.Wrap(err, "initializing Alertmanager object failed")
 		}
 
+		// Use console URL as Alertmanager ExternalURL since UI access to AlertManager is disabled.
+		consoleRoute, err := t.client.GetRouteWithNameAndNameSpace(ctx, "console", "openshift-console")
+		if err != nil {
+			return errors.Wrap(err, "getting console route failed")
+		}
```

Comment:
This is probably superfluous? `GetRouteUrl` does a get with the passed route name and namespace, so its sufficient to pass a route object with those fields set I think.

Reply:
ok, It will be changed back to GetRouteUrl for simplicity.

Reply:
I wonder which starts first, the CMO or console. The test show it fails to find the route console in namespace openshift-console.